acceptance_test.go

```diff
@@ -498,6 +482,14 @@ func (e *Environment) Test_Raciness(t *testing.T) {
 	time.Sleep(time.Second * 10)
 }
 
+func testResponseSuccess(t *testing.T, resp *http.Response) {
+	t.Helper()
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+		body, _ := ioutil.ReadAll(resp.Body)
```

Comment:
I think that this would panic if you pass in nil, so should we nil check against `resp` and `resp.Body` in here beforehand? Alternatively, we could skip reading the body if we only care about validating the response code.

Reply:
I feel like the body is useful in the event of a failure. I would prefer it to be more verbose. I will include the nil checks, thanks!